### DISCUSSION TITLE
Add Explorer Patcher options to the taskbar context menu

### DISCUSSION THREAD

**Discussion by Anix-x** [valinet/ExplorerPatcher#108] [Ideas]
It is hardly noticeable and accessible in Win+X menu.

**Comment by valinet** [comment]
I think it is fine where it is. It is less prone to bugs, it is debugged already and sits fine there. It's accessible enough. With it in 2 places, it becomes too bloated anyways. My vision is to keep the thing as untouched as possible, I kind of believe even there is too much, if it were only for me, I'd just manually launch it using `rundll32.exe`. I want the patcher to feel Windows native. Adding stuff to the menus takes away from that.

Please open a single discussion where you post your wish list. Thanks.

**Comment by Gaurav-Original-ClassicShellTester** [comment]
> It is hardly noticeable and accessible in Win+X menu.

I made a shortcut in my Open Shell Start menu to it:
Rundll32 %windir%\dxgi.dll,ZZGUI

That way, the GUI also doesn't close when Explorer is restarted. Best thing ever. :))

**Comment by valinet** [comment]
Yeah, that's indeed one way to do it. Since the last version, the GUI also survives Explorer restarts when invoked via `Win`+`X` > Properties. It is now hosted out of process.
